src/server.py

```diff
@@ -543,6 +472,7 @@ def buildbot_name(platform, buildtype, jobname, branch):
     platform_map['windowsxp'] = "Windows XP 32-bit"
     platform_map['windows7-32'] = "Windows 7 32-bit"
     platform_map['windows8-64'] = "Windows 8 64-bit"
+    platform_map['windows7-32-vm'] = "Windows 7 VM 32-bit"
```

Comment:
this platform is created because we are trying to run all our windows tests on vm instead of hardware and we need to hide certain jobs as they were confusing developers.